icepyx/core/query.py

```diff
@@ -1113,7 +1065,7 @@ def order_granules(self, verbose=False, subset=True, email=False, **kwargs):
                 self.subsetparams(**kwargs),
                 verbose,
                 subset,
-                session=self._session,
+                session=self.session,
```

Comment:
```suggestion
                session=self.auth.session,
```

Reply:
This isn't needed. Same reasoning as [previous comment](https://github.com/icesat2py/icepyx/pull/435#discussion_r1301642848).